tsl/test/expected/cagg_query_using_merge.out

```diff
@@ -243,28 +243,25 @@ select * from mat_m1 order by location, timec asc;
 
 :EXPLAIN
 select * from mat_m1 where timec > '2018-10-01' order by timec desc;
-                                                                                                         QUERY PLAN                                                                                                         
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
- Sort
-   Output: _hyper_2_4_chunk.location, _hyper_2_4_chunk.timec, _hyper_2_4_chunk.minl, _hyper_2_4_chunk.sumt, _hyper_2_4_chunk.sumh
-   Sort Key: _hyper_2_4_chunk.timec DESC
-   ->  Append
```

Comment:
Do you know why Postgres doesn't do MergeAppend here? It's not as good as the new Ordered Append plan, but might be another missing optimization to explore.